src/genie/libs/parser/dell/os6/tests/ShowTerminalLength/equal/golden_output_expected.py

```diff
@@ -0,0 +1 @@
+expected_output = {'length': 24}
```

Comment:
below might be easier to understand even only from parsed_output.
```suggestion
expected_output = {'terminal_length': 24}
```